### PR TITLE
RAxML UX improvements from Michael Buhl

### PR DESCRIPTION
**Pull request by bgruening**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/raxml/raxml.xml [hunk 1]
```diff
@@ -117,7 +117,8 @@ fi;
 #end if
     ]]></command>
     <inputs>
-        <param name="infile" argument="-s" type="data" format="fasta,phylip" label="Source file" />
+        <param name="infile" argument="-s" type="data" format="fasta,phylip" label="Source file with aligned sequences"
+            help="At least for aligned genomes are needed for RAxML." />
```
Comment (bernt-matthias) [inline tools/raxml/raxml.xml:121]: Somehow the sentence in the help attribute seems odd. Also, its a mandatory argument, so why: "are needed at least for"  (they are always needed, or?)

Comment (bgruening) [reply tools/raxml/raxml.xml]: 4 genomes ... sorry, spelling mistake, does it make more sense now?

**Comment by bernt-matthias** [comment]
Anything else you want to change?

**Comment by bgruening** [comment]
Nope, this is ready to go I think :)

**Pull request merged**